cms/static/js/spec/views/pages/index_spec.js

```diff
@@ -101,7 +101,7 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/spec_helpers
                     var values_with_blank = values.slice();
                     values_with_blank[i] = '';
                     fillInLibraryFields.apply(this, values_with_blank);
-                    expect($('.create-library li.field.text input[value=]').parent()).toHaveClass('error');
+                    expect($('.create-library li.field.text input').parent()).toHaveClass('error');
```

Comment:
Note to reviewers: the previous syntax was incorrect. We don't really need to search for inputs with a particular value because in this case they all have the same (empty) value.